### PR TITLE
Olwa/dev

### PR DESCRIPTION
**Pull request by owalk**
I've improved the collision logic by a good amount, but it is still not working. We will need to look into it on Monday

### CODE REVIEW

**Comment by owalk** [comment]
I have more commits in progress and might not finish today, so this is now a draft.

**Comment by owalk** [comment]
This is probably good. I haven't done a very thorough self-review yet tho, so if there is time later tonight I will.
Otherwise, we can do it together on Monday

File: game/entities/damsel.py [hunk 1]
```diff
@@ -109,23 +109,23 @@ def move(self, speed):
...
+                self.compass.y = -1
             else:
-                self.direction.y = 1
+                self.compass.y = 1
             self.timer = 0
 
         # prevent diagonal moving from increasing speed
-        if self.direction.magnitude() != 0:
-            self.direction = self.direction.normalize()
+        if self.compass.magnitude() != 0:
```
Comment (owalk) [inline game/entities/damsel.py]: Normalize is not needed anymore. Movement logic changed. Add a new issue!!

File: game/entities/skeleton.py [hunk 2]
```diff
@@ -119,23 +119,23 @@ def move(self, speed):
...
+                self.compass.y = -1
             else:
-                self.direction.y = 1
+                self.compass.y = 1
             self.timer = 0
 
         # prevent diagonal moving from increasing speed
-        if self.direction.magnitude() != 0:
-            self.direction = self.direction.normalize()
+        if self.compass.magnitude() != 0:
```
Comment (owalk) [inline game/entities/skeleton.py]: Remove normalization. Add as a new issue!

**Review by Sean-Nishi — APPROVED** [review]

**Pull request merged**